tools/webhook-ingestion-pipeline/webhook_endpoint/main.py

```diff
@@ -51,6 +55,22 @@ def handle_invalid_usage(error):
     response.status_code = error.status_code
     return response
 
+
+def _append_metadata(json_data):
+    """ Appends necessary _metadata fields s
+        for downstream processing
+        and event splitting
+    """
+    default_metadata = {'@uuid': str(uuid.uuid4()),
+                        '@timestamp': datetime.utcnow().isoformat()}
+    if isinstance(json_data, str):
+        json_data = json.loads(json_data)
+
+    if '_metadata' not in json_data.keys():
```

Comment:
unecessary call to `dict.keys`.
```suggestion
    if '_metadata' not in json_data:
```